internal/functions/property_display_name.go

```diff
@@ -0,0 +1,13 @@
+package functions
+
+import "github.com/theunrepentantgeek/crddoc/internal/model"
+
+// propertyDisplayName returns the appropriate name for a property based on the configuration.
+// If UseGoFieldNames is true, returns the Go field name.
+// Otherwise, returns the serialized name (from JSON/YAML tags).
+func (f *Functions) propertyDisplayName(property *model.Property) string {
+	if f.cfg != nil && f.cfg.UseGoFieldNames {
+		return property.Field
+	}
+	return property.Name
+}
```

Comment:
Files must always finish with an EoLn - failing to do this can cause problems when the source files are consumed by some tools. Fix this and any other cases in this PR.